# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1290128489 18000
#      Thu Nov 18 20:01:29 2010 -0500
# Node ID da314c58f0b9bc9c4e05e59118ee803fe908c470
# Parent  5dd8e80fb6c198a6c386526469848c2a6a86999b
__go_draw_axes__.m: Fix bug setting zscale to yscale (31693).

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-11-18  Ben Abbott <bpabbott@mac.com>
+
+	* plot/__go_draw_axes__.m: Fix bug setting zscale to yscale (31693).
+
 2010-11-18  Christos Dimitrakakis  <christos.dimitrakakis@gmail.com>
 
 	* statistics/distributions/betapdf.m: Use lgamma to compute
 	normalising constant in log space in order to handle large
 	parameters a and b.  Ensure correct values at x == 0 or x == 1.
 
 2010-11-18  Ben Abbott <bpabbott@mac.com>
 
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -1922,17 +1922,17 @@ function do_tics (obj, plot_stream, ymir
                obj.interpreter, obj.yscale);
     do_tics_1 ("manual", [], "off", obj.yticklabelmode, obj.yticklabel,
                obj.ycolor, "y2", plot_stream, ymirror, mono, "border",
                "", "", fontname, fontspec, obj.interpreter, obj.yscale);
   endif
   do_tics_1 (obj.ztickmode, obj.ztick, obj.zminortick, obj.zticklabelmode,
              obj.zticklabel, obj.zcolor, "z", plot_stream, true, mono,
              "border", obj.tickdir, ticklength, fontname, fontspec,
-             obj.interpreter, obj.yscale);
+             obj.interpreter, obj.zscale);
 endfunction
 
 function do_tics_1 (ticmode, tics, mtics, labelmode, labels, color, ax,
                     plot_stream, mirror, mono, axispos, tickdir, ticklength,
                     fontname, fontspec, interpreter, scale)
   persistent warned_latex = false;
   if (strcmpi (interpreter, "tex"))
     for n = 1 : numel(labels)
